plugins/kubernetes/app/models/kubernetes/release_doc.rb

```diff
@@ -20,6 +20,10 @@ def client
       deploy_group.kubernetes_cluster.client
     end
 
+    def job?
+      deploy_yaml.resource_name == 'job'
```

Comment:
resource_name? Maybe it's better (in a future patch) to use "resource_kind" to avoid confusions with the "name" metadata of the object?